clients/privacy-center/pages/index.tsx

```diff
@@ -33,7 +35,7 @@ const Home: NextPage = () => {
     successHandler: privacyModalSuccessHandler,
   } = usePrivacyRequestModal();
 
-  const {
+  let {
```

Comment:
I'm not super happy with having to do this for every var assigned in that dict based on needing to override just one